docs/codeql/codeql-language-guides/codeql-for-javascript.rst

```diff
@@ -13,7 +13,6 @@ Experiment and learn how to write effective and efficient queries for CodeQL dat
    codeql-library-for-typescript
    analyzing-data-flow-in-javascript-and-typescript
    using-flow-labels-for-precise-data-flow-analysis
-   customizing-library-models-for-javascript
```

Comment:
Can you confirm that even with this file no longer being linked to, the rst is still compiled?

Reply:
Actually the whole build fails now because Sphinx detects that the file is inaccessible.

How about just keeping the links public? I thought the beta notice at the top of the page would be good enough.

Reply:
That's unfortunate. I don't understand how the sphinx build works, so not sure what's easy to do. If there's an easy way to do this, I'd still like to, but not worth spending any extra effort for this.

Reply:
Asked about this on [#docs-content](https://github.slack.com/archives/C0E9DK082/p1678120302633849) and seems like our best option is to keep as-is with the beta notice.